cmd/analyzer/main.go

```diff
@@ -87,142 +55,181 @@ func main() {
 	logrusErrLog.SetOutput(os.Stderr)
 	errLog := logrusr.New(logrusErrLog)
 
-	err := validateFlags()
-	if err != nil {
-		errLog.Error(err, "failed to validate flags")
-		os.Exit(1)
-	}
+	rootCmd := &cobra.Command{
+		Use:   "analyze",
+		Short: "Tool for working with analyzer-lsp",
+		PreRunE: func(c *cobra.Command, args []string) error {
+			err := validateFlags()
+			if err != nil {
+				errLog.Error(err, "failed to validate flags")
```

Comment:
nit: can we `return err` here? and return nil after the if block? just making sure if we make changes in future, it doesn't go unnoticed that err could be non-nil after the if block